### PR TITLE
Use ngx_http_nchan_module as the module name

### PR DESCRIPTION
**Pull request by ctrochalakis**
Follow nginx common practices for module names.

### CODE REVIEW

**Comment by ctrochalakis** [comment]
Note that, it builds ok, but I have not tested loading it into nginx.

**Comment by slact** [comment]
Two things about this:

First, while currently nchan is internally only an http module, this will not be the case for long. As nginx 1.9.13 now supports UDP via the stream module, I am already working on dtls-srtp support and other non-http-based features. This is why i've taked the _http_ out of the module name.

As for the ngx_ prefix, that's easier. The nchan code does not follow the nginx style conventions (whitespace, brace placement, etc), whcih is why I thought it appropriate to omit said prefix from most of the code and the source filenames. However, it looks like nchan is the only one that does this for the module name itself, and I can certainly toe the line in regards to that.

Tl;DR: I'll make it ngx_nchan_module
